### PR TITLE
Updated SDK

### PR DESCRIPTION
**Pull request by flashburst**
- Updated SDK version
- Added policy purchase feature for diversified cover (https://github.com/neptune-mutual-blue/app.neptunemutual.com/pull/399)
- Get reporting info from provider
- Fixed division by zero error

### CODE REVIEW

**Review by naicky-neptune — APPROVED** [review]

**Review by al-neptune — APPROVED** [review]

File: src/common/Cover/CoverStatsContext.jsx [hunk 1]
```diff
@@ -26,10 +27,28 @@ export const CoverStatsProvider = ({ coverKey, children }) => {
...
+    productStatus,
     totalPoolAmount,
     availableLiquidity,
-  } = useFetchCoverStats({ coverKey, productKey: safeFormatBytes32String("") });
+  } = useFetchCoverStats({
+    coverKey,
+    productKey: productKey || safeFormatBytes32String(""),
+  });
+
+  console.log({
```
[the remark this answers is not in the document]
Comment (flashburst) [reply src/common/Cover/CoverStatsContext.jsx]: Done

File: src/modules/cover/purchase/index.jsx [hunk 2]
```diff
@@ -18,20 +18,31 @@ import { t, Trans } from "@lingui/macro";
...
+    isBasket ? "basket" : "standalone"
+  );
+
+  let coverInfo;
+  if (!isBasket) {
+    coverInfo = getInfoByKey(coverKey);
+  }
+  if (isBasket) {
+    coverInfo = getBasketInfoByKey(coverKey, productKey);
+  }
```
[the remark this answers is not in the document]
Comment (flashburst) [reply src/modules/cover/purchase/index.jsx]: Done

File: src/modules/cover/purchase/index.jsx [hunk 3]
```diff
@@ -18,20 +18,31 @@ import { t, Trans } from "@lingui/macro";
...
   const { cover_id, product_id } = router.query;
   const coverKey = safeFormatBytes32String(cover_id);
   const productKey = safeFormatBytes32String(product_id || "");
-  const { getInfoByKey } = useCovers();
-  const coverInfo = getInfoByKey(coverKey);
   const { liquidityTokenDecimals, liquidityTokenSymbol } = useAppConstants();
-
   const { info } = useMyLiquidityInfo({ coverKey });
+
+  const isBasket = typeof product_id !== "undefined" ? true : false;
```
[the remark this answers is not in the document]
Comment (flashburst) [reply src/modules/cover/purchase/index.jsx]: Done

File: src/common/Cover/CoverStatsContext.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (flashburst) [reply src/common/Cover/CoverStatsContext.jsx]: Done

**Review by savinnpm — APPROVED** [review]

**Pull request merged**